sklearn/metrics/_plot/roc_curve.py

```diff
@@ -14,25 +21,53 @@ class RocCurveDisplay(_BinaryClassifierCurveDisplayMixin):
 
     It is recommend to use
     :func:`~sklearn.metrics.RocCurveDisplay.from_estimator` or
-    :func:`~sklearn.metrics.RocCurveDisplay.from_predictions` to create
+    :func:`~sklearn.metrics.RocCurveDisplay.from_predictions` or
+    :func:`~sklearn.metrics.RocCurveDisplay.from_cv_results` to create
     a :class:`~sklearn.metrics.RocCurveDisplay`. All parameters are
     stored as attributes.
 
-    Read more in the :ref:`User Guide <visualizations>`.
+    For more about the ROC metric, see :ref:`roc_metrics`.
+    For more about scikit-learn visualization classes, see :ref:`visualizations`.
 
     Parameters
     ----------
-    fpr : ndarray
-        False positive rate.
-
-    tpr : ndarray
-        True positive rate.
-
-    roc_auc : float, default=None
-        Area under ROC curve. If None, the roc_auc score is not shown.
-
-    estimator_name : str, default=None
-        Name of estimator. If None, the estimator name is not shown.
+    fpr : ndarray or list of ndarrays
+        False positive rates. Each ndarray should contain values for a single curve.
+        If plotting multiple curves, list should be of same length as
+        and `tpr`.
+
+    tpr : ndarray list of ndarrays
+        True positive rates. Each ndarray should contain values for a single curve.
+        If plotting multiple curves, list should be of same length as
+        and `fpr`.
+
+    roc_auc : float or list of floats, default=None
+        Area under ROC curve, used for labeling each curve in the legend.
+        If plotting multiple curves, should be a list of the same length as `fpr`
+        and `tpr`. If `None`, individual ROC AUC scores are not shown. See
+        `roc_auc_aggregate` for alternative.
+        If `name` and `roc_auc_aggregate` are also `None` no legend is added.
+
+    roc_auc_aggregate : tuple(float), default=None
+        ROC AUC mean and standard deviation. An alternative to `roc_auc` when
+        plotting multiple curves and a single legend entry showing ROC AUC mean and
+        standard deviation for all curves is desired.
+        If `True`, `name` cannot be a list of length >1.
+
```

Comment:
I don't think that we need to introduce this parameter. Similar to `show_aggregate_score`, I think it's redundant and brings to much coupling between parameters.

Instead, I think that deciding on showing an aggregate or not should be done automatically in `plot`, depending on whether `curve_name` or `roc_line_kwargs` are lists or not.